tests/gcp/hooks/test_bigquery_system.py

```diff
@@ -31,51 +30,26 @@ class BigQueryDataframeResultsSystemTest(unittest.TestCase):
     def setUp(self):
         self.instance = hook.BigQueryHook()
 
-    @mock.patch(
-        'airflow.gcp.hooks.base.GoogleCloudBaseHook.project_id',
-        new_callable=mock.PropertyMock,
-        return_value=None
-    )
-    def test_output_is_dataframe_with_valid_query(self, mock_project_id):
+    def test_output_is_dataframe_with_valid_query(self):
```

Comment:
Is it related to this PR?

Reply:
Actually it is not. I will rollback these changes and create separate PR for this change.